scalableminds/webknossos
# Convert to wkw unit support

WK part to enable passing the user-provided unit to the wkw conversion job. 

Please note that I tried to improve the typing and naming of the upload view in the frontend :). This just slipped in as an improvement.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Better test locally in conjunction with the new worker pr [#3666](https://github.com/scalableminds/voxelytics/pull/3666)
- Set up a work with the version of the linked vx pr. (`yarn enable-jobs`)
- Upload a dataset that needs to be converted and select a unit other than nm
- After the upload and conversion are finished try viewing the dataset and check that the unit is according to what you configured during the upload.

### Issues:
- fixes [#7940](https://github.com/scalableminds/webknossos/issues/7940)

------
(Please delete unneeded items, merge only when none are left open)
